### ISSUE TITLE
dap doesn't delete "all" paragraph

### ISSUE THREAD

**Issue by mikebenfield** [VSCodeVim/Vim#1928]
* Click *thumbs-up* 👍 on this issue if you want it!
* Click *confused* 😕 on this issue if not having it makes VSCodeVim unusable. 

The VSCodeVim team prioritizes issues based on reaction count. 

--------

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

BUG REPORT

**Environment**:

- **VSCode Version**: 1.14.1
- **VsCodeVim Version**: 0.9.0
- **OS**: Mac OS 10.12.4

**What happened**:

Given:
```
abcdef

■hijkl
ghijkl
ghijkl

mnopqr
```

I press `dap` and get
```
abcdef


mnopqr
```

(with two empty lines).

**What did you expect to happen**: 

I should get

```
abcdef

mnopqr
```
(with one empty line).

**Comment by brandonbloom** [comment]
This was fixed by #1996

**Comment by Chillee** [comment]
Awesome!
